styles/styles.scss

```
@@ -112,6 +112,26 @@ img {
   object-fit: cover;
 }
 
+.usa-alert.usa-alert--info,
+.usa-alert.usa-alert--info .usa-alert__body {
+  background: transparent;
+}
+
+.usa-alert.usa-alert--info {
+  @include u-bg("yellow-5v");
+  @include u-border-left('accent-warm');
+}
+
+.usa-alert.usa-alert--info.usa-alert--slim,
+.usa-alert.usa-alert--info.usa-alert--slim .usa-alert__body {
+  background: transparent;
+}
+
+.usa-alert.usa-alert--info.usa-alert--slim {
+  @include u-bg("yellow-5v");
+  @include u-border-left('accent-warm');
+}
+
```

You can combine like this:
```suggestion
.usa-alert.usa-alert--info,
.usa-alert.usa-alert--info .usa-alert__body,
.usa-alert.usa-alert--info.usa-alert--slim,
.usa-alert.usa-alert--info.usa-alert--slim .usa-alert__body {
  background: transparent;
}

.usa-alert.usa-alert--info,
.usa-alert.usa-alert--info.usa-alert--slim {
  @include u-bg("yellow-5v");
  @include u-border-left("accent-warm");
}
```